fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/ItemTagGenerator.java

```diff
@@ -268,8 +268,8 @@ private void generateOtherTags() {
 		getOrCreateTagBuilder(ConventionalItemTags.WOODEN_RODS)
 				.add(Items.STICK);
 		getOrCreateTagBuilder(ConventionalItemTags.RODS)
-				.addOptionalTag(ConventionalItemTags.WOODEN_RODS)
-				.add(Items.BLAZE_ROD);
+				.add(Items.BLAZE_ROD)
```

Comment:
Wooden rods are way more representative of this tag than blaze rods.

Reply:
It's from the Neo PR which from forge had `forge:rods`, `forge:rods/blaze`, `forge:rods/wooden`. To not break the Neo ports, I pulled that to `c:rods`, `c:rods/blaze`, `c:rods/wooden`. Then I saw several fabric mods (about 6) were trying to query `c:wooden_rods` so there was demand on fabric side for this so I ported  `c:rods` and `c:rods/wooden`, leaving out `c:rods/blaze`as I did not find anyone trying to use the blaze rod tag but still added it to rods tag to match neo's.

I guess historically, forge tried to label sticks and blaze rods as a single category. Not sure if it really survives the test of time hmm. I don't like the name rod and rods/blaze seems utterly useless. A simple `c:sticks` tag to replace these three rods tags may be better overall

Reply:
Nvm. ignore everything I said. Gotta keep rods but yeah, wooden should be first due to high usage 
![image](https://github.com/FabricMC/fabric/assets/40846040/7c2da7d0-bd5a-47ea-8b23-c50e76ae2a0f)